sdk/storage/azure-storage-common/CMakeLists.txt

```diff
@@ -56,22 +66,34 @@ set(
 add_library(azure-storage-common ${AZURE_STORAGE_COMMON_HEADER} ${AZURE_STORAGE_COMMON_SOURCE})
 
 # make sure that users can consume the project as a library.
-add_library(azure::storage::common ALIAS azure-storage-common)
+add_library(Azure::azure-storage-common ALIAS azure-storage-common)
+
+target_include_directories(
+  azure-storage-common
+    PUBLIC
+      $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc>
+      $<INSTALL_INTERFACE:include>
+      ${azure-core-cpp_INCLUDE_DIRS}
+)
+
+target_include_directories(azure-storage-common INTERFACE ${LIBXML2_INCLUDE_DIRS})
+target_link_libraries(azure-storage-common PUBLIC Azure::azure-core)
+target_link_libraries(azure-storage-common INTERFACE Threads::Threads ${LIBXML2_LIBRARIES})
 
-target_include_directories(azure-storage-common PUBLIC inc ${LIBXML2_INCLUDE_DIRS})
-target_link_libraries(azure-storage-common Threads::Threads azure-core ${LIBXML2_LIBRARIES})
 if(MSVC)
-    target_link_libraries(azure-storage-common bcrypt)
+    target_link_libraries(azure-storage-common INTERFACE bcrypt)
```

Comment:
bcrypt is a dependency, why is it not listed in the package dependencies?

Reply:
bcrypt is a Win API header and not a package: https://docs.microsoft.com/en-us/windows/win32/api/bcrypt/
From the perspective of vcpkg, it is not a dependency we are bringing.